src/commonMain/kotlin/app/cash/turbine/FlowTurbine.kt

```diff
@@ -190,6 +190,15 @@ public interface FlowTurbine<T> {
    */
   public suspend fun awaitItem(): T
 
+  /**
+   * Assert that [count] item events were received and ignore them.
+   * If not enough events have been received, this function will suspend for up to [timeout].
```

Comment:
The current behavior suspend for up to the timeout _for each item_. Should the timeout apply to the whole operation? Or should we fix the docs?

Reply:
I think it's fine to do it per item. 
Will fix the docs.